deploy/common/k8sbased.sh

```diff
@@ -712,12 +712,14 @@ function launch_sensor {
     fi
 
     # If deploying with chaos proxy enabled, patch sensor to add toxiproxy proxy deployment
-    if [[ "$CHAOS_PROXY" == "true" ]]; then
+    if [[ -n "${ROX_CHAOS_PROFILE}" ]]; then
```

Comment:
Do we have a guarantee that `ROX_CHAOS_PROFILE` is empty in all cases when we don't want the proxy setup?

Reply:
I believe we don't set it anywhere so it should be empty unless the user specifically sets the variable.

Reply:
Okay, let's look what CI would say (except the ocp ones that are affected today).